### DISCUSSION TITLE
Any plans for a vimscript/lua API?

### DISCUSSION THREAD

**Discussion by IndianBoy42** [mg979/vim-visual-multi#232] [Ideas]
Patterns are cool, using treesitter or LSP to create selections could be even cooler. A function that can be called with a `line, col` or `line,col,line,col` to add a cursor/selection would probably enable a lot of stuff

**Comment by mg979** [comment]
You can do that already I guess, if you make the function yourself.

**Reply by IndianBoy42** [reply]
How would I make that function? Other than moving the cursor and using feedkeys, to just simulate a human

**Reply by mg979** [reply]
No idea sorry.
